distributed/deploy/tests/test_spec_cluster.py

```diff
@@ -131,6 +132,33 @@ class MyCluster(SpecCluster):
         assert len(cluster.workers) == 2
 
 
+@pytest.mark.asyncio
```

Comment:
It looks like this test takes a while (longer than a second).  If we can, it would be good to reduce the time here.  Most tests run in under a second, and it's nice to keep the test suite running reasonably fast.  This may not be possible given the nature of the test, but anything that you can do here would be welcome.

Additionally, we should mark this test with `@pytest.mark.slow`

Reply:
I've tried to cut down the time taken to run the test but couldn't get it below ~4s unfortunately so I've marked it as slow!